packages/runtime/src/state/actions.ts

```diff
@@ -72,10 +72,6 @@ export const ActionTypes = {
   MAKESWIFT_CONNECTION_INIT: 'MAKESWIFT_CONNECTION_INIT',
   MAKESWIFT_CONNECTION_CHECK: 'MAKESWIFT_CONNECTION_CHECK',
 
-  CHANGE_PATHNAME: 'CHANGE_PATHNAME',
-  CHANGE_PATHNAME_START: 'CHANGE_PATHNAME_START',
-  CHANGE_PATHNAME_COMPLETE: 'CHANGE_PATHNAME_COMPLETE',
```

Comment:
Unfortunately, these are also used in the builder (the entire `ActionTypes` object is 😢), so is the plan to remove that code from the builder?